src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.shared.cs

```diff
@@ -16,6 +16,14 @@ public partial class Snackbar : ISnackbar
 	/// </summary>
 	public Snackbar()
 	{
+		if (OperatingSystem.IsWindows() && !Options.ShouldEnableSnackbarOnWindows)
+		{
+			throw new InvalidOperationException($"Additional setup is required in the Package.appxmanifest file to enable {nameof(Snackbar)} on Windows. Additonally, `{nameof(AppBuilderExtensions.UseMauiCommunityToolkit)}(options => options.{nameof(Options.SetShouldEnableSnackbarOnWindows)}({true.ToString().ToLower()});` must be called to enable Snackbar on Windows. See the Platform Specific Initialization section of the {nameof(Snackbar)} documentaion for more information: https://learn.microsoft.com/dotnet/communitytoolkit/maui/alerts/snackbar")
```

Comment:
Please use Boolean TrueString instead of bool.ToString

Reply:
Cool! I hadn't heard of `bool.TrueString` before. 

The problem is that `bool.TrueString` returns `True` with an uppercase `T`, so I kept `.ToLower()`. 

We could also just hardcode the text `true` instead. I didn't mean to make this more complicated than it is 😆